pkgs/by-name/an/animdl/package.nix

```diff
@@ -55,12 +56,12 @@ python3Packages.buildPythonApplication {
 
   doCheck = true;
 
-  meta = with lib; {
+  meta = {
     description = "Highly efficient, powerful and fast anime scraper";
     homepage = "https://github.com/justfoolingaround/animdl";
-    license = licenses.gpl3Only;
+    license = lib.licenses.gpl3Only;
     mainProgram = "animdl";
-    maintainers = with maintainers; [ passivelemon ];
-    platforms = [ "x86_64-linux" ];
+    maintainers = with lib.maintainers; [ passivelemon ];
+    platforms = lib.platforms.linux ++ lib.platforms.darwin;
```

Comment:
```suggestion
```
https://nixos.org/manual/nixpkgs/stable/#contributing

Reply:
Thank you for your detailed review, as usual. I always learn some new nuance.